tinygrad/tensor.py

```diff
@@ -54,25 +54,19 @@ def __repr__(self):
 
 class Tensor:
   did_float_warning = False
-  default_gpu = False
   ops_cpu, ops_gpu = {}, {}
 
   def __init__(self, data, gpu=None, requires_grad=True):
-    if gpu is None:
-      gpu = Tensor.default_gpu
-    if isinstance(data, list):
-      data = np.array(data, dtype=np.float32)
-    elif GPU and isinstance(data, GPUBuffer):
-      self.gpu = True
+    self.gpu = False
```

Comment:
default_gpu gone?

Reply:
`default_gpu` is never set `True` unless the user at runtime sets `Tensor.default_gpu = True`. Perhaps this was supposed to read from the system environment(`export GPU=true`)?

But that being said, setting `default_gpu=True` fails the tests anyway. This is due to the way `.cuda_` handles conversions. 

For this reason, I removed it and if support needs to be added again, we should refactor `.cuda_` and `.cuda()`.

Reply:
I also believe that it would be better in the future to add default GPU support in the `kwargs` of the `__init__`.

I.e.
``` python
def __init__(self, data, gpu=GPU, requires_grad=True):
```
Or something like that.